cpp/serve/model.cc

```diff
@@ -94,6 +94,20 @@ class ModelImpl : public ModelObj {
     }
   }
 
+  ObjectRef ImageEmbed(const NDArray& image, ObjectRef* dst, int offset) final {
+    CHECK(ft_.image_embed_func_.defined()) << "`image_embed` function is not found in the model. ";
+    auto image_dref_or_nd = ft_.CopyToWorker0(image, "image", image.Shape());
```

Comment:
Here we want to pass the maximum possible shape in the third argument, through which we reserve the NDArray to the maximum possible size when allocating. Do all the images of the image embedding func have the same shape? If this is always true, we can pass `image.Shape()`. Or otherwise, we need to pass the maximum shape.

Reply:
Yes, in the `get_image_from_url` function, we process the image to be of a fixed size. This image is then passed to the model